meetings/2019-10-29.md

```diff
@@ -0,0 +1,95 @@
+# Code Curious Elixir Meeting 2019-10-29
+
+## Attendees
+
+- Jens (moderating)
+- Nicolo
+- Markus
+- Max
+- Tobi
+- Daniel
+
+## Topics
+
+- recap last TODOs
+- decide on workshop and audience
+- find date for workshop
+
+## Organizational matters
+
+- low attendance: check who's interested in coaching and organizing
+  - people are sick
+  - we are enough people to organize the workshop
+  - once we know about the workshop: make call for coaches
+- early Doodle and clear announcement helps for attendance
+- the work on the guide is done via PRs (merge whenever it makes sense)
+
+## Decide on workshop
+
+### Which audience
+
+- start with intermediate:
+  - works best for first workshop
+  - quick vote: we start with intermediate
+- "intermediate" means confident with with computers
+  - this of course includes juniors
+
+### Which content
+
+- recap:
+  - nerves project (we stopped looking at it)
+  - snake (people seem to like it)
+  - elixir-girls chat app (only looked at it briefly)
+    - we did not look into their intermediate guide
+- round of votes:
+  - snake fun and entertaining
+  - snake lots of Elixir good to start
+  - snake has visual aspects
+  - snake tutorial has "natural steps" (works great for guide)
+  - snake does not work for finding a job, but valid for tutorial
+  - hardware project: makes sense for the future, but too much orga effort for now
+  - elixir girls app: we would need to update the guide (no ETS, less JS)
+    - pro: great for lots of elixir and no phoenix
+    - pro: has lots of the language features in it
```

Comment:
Is it possible that these two points are referring to the Snake Tutorial?

Reply:
In theory yes, but they were mentioned when referring to the Slack clone tutorial.